package.yaml

```diff
@@ -20,8 +20,7 @@ ghc-options:
   - -Wredundant-constraints
   - -threaded
 
-data-files:
-- rad/**/*.rad
+data-dir: rad
```

Comment:
This does not seem to work for me. `stack build` will not copy the files into the right location.

Reply:
Explains why CI was failing!